supporter-product-data/cloudformation/validate-cloudformation.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+aws --region eu-west-1 cloudformation validate-template --template-body file://cfn.yaml --profile membership
```

Comment:
is something missing from the file// url here?  or does it require you to CD to the right directory first? could add `dirname $0` to the url if that's the case

Reply:
The script is in the same directory as the cfn file